bcda/auth/provider.go

```diff
@@ -28,18 +30,18 @@ func SetProvider(name string) {
 		case Alpha:
 			providerName = name
 		default:
-			log.Infof(`Unknown providerName %s; using %s`, name, providerName)
+			log.Infof(`unknown providerName %s; using %s`, name, providerName)
```

Comment:
It looks like this and some other log messages were switched to not be capitalized, but I think that is a pattern we want to maintain (not to be confused with error text where this format is the standard practice)